### DISCUSSION TITLE
Messages at print start after update do 1.3.3.8

### DISCUSSION THREAD

**Discussion by KlausStoertebeker** [Guilouz/Creality-Helper-Script-Wiki#490] [Issues]
Hello,
I updated my Creality K1 Max to system software 1.3.3.8. Since then I always get the following (error?) messages when starting a print:

`20:55:59 File opened:5_squares_0.2mm_height_bed_leveling_test_ABS_2m35s.gcode Size:38851`
`20:55:59 File selected`
`20:58:24 // {"code":"key61, "msg":"Unknown command:BED_LEVELING_START", "values": ["BED_LEVELING_START"]}`
`20:58:24 !! {"code":"key95","msg":"Must home axis first: 0.000 0.000 1.266 [0.000]", "values":[0.000, 0.000, 1.266, 0.000]}`

New to me is that the print head isn't homing automatically (I think it does with system software 1.3.3.5) and that bed leveling is not possible because the command/gcode-script `BED_LEVELING_START` is unknown. This is strange because I can start a bed leveling via system self test.
Is there someone who has the same issue and maybe a solution for this?

Regards from Dresden

"Klaus"

**Comment by alientmpl** [comment]
I'm not sure, but using the home button on the main page helped me
![home](https://github.com/Guilouz/Creality-K1-Series/assets/60255987/29e08a7b-8155-4ac2-980c-d795728208b8)

**Reply by KlausStoertebeker** [reply]
Hello alientempl,
thanks for reply but I tried that and it works for me, too. But I have to do it MANUALLY whereas it was done automatically before when starting a print.
And why is the z-offset not zero but 1.26 mm? I haven't changed the setup anyway except doing a bed leveling during a manually started self test - which was reported ok...
